daphne/src/vdaf/mod.rs

```diff
@@ -364,7 +366,7 @@ where
     S: Serializer,
     T: Encode,
 {
-    s.serialize_str(&hex::encode(x.get_encoded()))
+    s.serialize_str(&hex::encode(x.get_encoded().expect("XXX")))
```

Comment:
@mendess how should we handle this encoding failure? Previously we would have panicked.

Reply:
`.map_err(|e| S::Error::custom(e))?)`